training/culture-and-process/PULL_REQUEST_GUIDANCE.md

```diff
@@ -0,0 +1,42 @@
+Pull Request Guidance
+===========================
+
+General guidance on creating a pull request
+
+### Pre reading
+
+- It is useful to have an understanding of GIT (https://git-scm.com/about) and branches as a way of working
+- Have a look at pull requests in the `dev-code-review` channel on slack
+- There is also this documentation on our contributing guidelines (https://github.com/ONSdigital/dp/blob/master/guides/CONTRIBUTING.md#development-work) which covers some aspects of creating pull requests. 
+
+### Pre requisites
+
+- In order to contribute you need to first step up a GPG key and configure git to sign all commits (https://github.com/ONSdigital/dp/blob/master/guides/GPG.md).
+
+### Steps
+
+To merge a feature branch into the develop or master/main branch it has to have an approved pull request. 
```

Comment:
To merge a feature branch into the `develop` or `master/main` branch it has to have an approved pull request. Just needs a backtick